Coralogix/Sources/CoralogixRum.swift

```diff
@@ -74,6 +93,7 @@ public class CoralogixRum {
                 .build())
         
         self.swizzle()
+        self.initializeAppLifeCycleInstrumentation()
```

Comment:
i think it should be like other instrumentations, that user can take it off/on in init
not everyone will want this

Reply:
here                                     
      instrumentations: [.navigation: true,
                                                              .mobileVitals: false,
                                                              .custom: true,
                                                              .errors: true,
                                                              .userActions: false,
                                                              .network: true,
                                                              .anr: true])

Reply:
it is it is just arm the register of the notification. inside each register i check if the user asked for that instrumentation